editor/modes/visual-editor/block.js

```diff
@@ -87,6 +87,9 @@ class VisualEditorBlock extends Component {
 		if ( this.props.isTyping ) {
 			document.addEventListener( 'mousemove', this.stopTypingOnMouseMove );
 		}
+
+		// Not Ideal, but it's the easiest way to get the scrollable container
+		this.editorLayout = document.querySelector( '.editor-layout__editor' );
```

Comment:
Any other ideas here?

Reply:
Multi-select also broke because of the same scroll container change. I've changed that so it's independent of the scroll position, but in this case you need the container to do a scroll... Maybe better to pass the container down as a prop?

Reply:
We can use the prop in block-list too then, and avoid calculating the dimension on scroll and mousemove.

Reply:
Popover achieved this (specifying DOM target) with a context provider, though I'm seeking to move away from it in favor of Slot-Fill:

https://github.com/WordPress/gutenberg/tree/master/components/popover#usage